op_crates/web/00_webidl.js

```diff
@@ -662,10 +662,40 @@
     };
   }
 
-  window.__bootstrap = window.__bootstrap || {};
+  function createPrimitiveEnum(name, opts) {
+    const Enum = new Set(opts.enumValues);
+
+    let defaultValue;
+
+    const hasDefault = "defaultValue" in opts;
+
+    if (hasDefault) {
+      defaultValue = opts.defaultValue;
+    }
+
+    const { converter } = ops;
```

Comment:
Where is ops defined?

Reply:
Oh, `opts`, missed a letter; I'll review the WebIDL spec you linked earlier and rewrite this anyway.